WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -21,7 +21,8 @@ onPage('registrations#edit_registrations', function() {
   showHideActions();
 
   $('button[value=delete-selected]').on("click", function(e) {
-    if(!confirm("Are you sure you want to delete the selected registrations?")) {
+    var $selectedRows = $registrationsTable.find("tr.selected");
+    if(!confirm("You are about to delete " + $selectedRows.length + " registrations.\nAre you sure you want to do it?")) {
```

Comment:
I actually like your last suggestion!

I changed the message, however I'll leave pluralization for later as it will easily be fixed when I'll internationalize strings in js.